backend/search/models.py

```diff
@@ -325,31 +319,28 @@ def get_results(self, from_number: int = 0, to_number: int = None) \
                 counts.append({
                     'component': result_obj.component.slug,
                     'number_of_results': result_obj.number_of_results,
-                    'completed': completed,
+                    'completed': result_obj.search_completed,
```

Comment:
result_obj.search_completed is a timestamp and not a boolean, but the frontend TypeScript expects a boolean, so I think we still need line 306. (I believe it is not used right now in the frontend though.) Naming is confusing though :(